pkg/plugins/platform/local.go

```diff
@@ -258,6 +258,15 @@ func (l *LocalApp) Start() error {
 	composePath := l.DockerComposeYAMLPath()
 	l.DockerEnv()
 
+	existing, err := l.FindContainerByType("web")
```

Comment:
I don't think this actually resolves the key problem here because it assumes that the competing site is running, which might or might not be the case. I understand that we're storing all our state info in container labels, which prevents us from a robust approach here. 

I'm not sure what the correct approach is here, but this doesn't fully solve https://github.com/drud/ddev/issues/136 - and of course not mentioned there are the facts that `ddev rm` can destroy a site from another directory, etc.  I imagine we have to have a conversation about how much of it is worth solving, and whether we can do it with our current level of config (mostly stashed on running containers, and nothing global.)